src/Button/UploadButton/UploadButton.tsx

```diff
@@ -18,40 +34,14 @@ import './UploadButton.less';
  * @class The UploadButton
  * @extends React.Component
  */
-class UploadButton extends React.Component {
+class UploadButton extends React.Component<UploadButtonProps> {
```

Comment:
Would it make sense to include an empty state interface here as well for the sake of completeness?

Reply:
No. I don't think we should do that.

Reply:
Alright.